docs/source/model_design/risk_factors.rst

```diff
@@ -531,9 +531,206 @@ Difference measures relating to the whole population tend to be more useful and
 Population Attributable Fraction (PAF)
 ++++++++++++++++++++++++++++++++++++++
 
+The population attributable fraction is a quantification of the proportion of 
+of a given cause outcome, such as cases, deaths, or DALYs, that could be 
+eliminated by removing a risk exposure.
+
+..todo::
+	Fill in PAF overview
+
+Bias in PAF Calculation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The PAF can be calculated using the following formula:
+
+.. math::
+	:label: eq1
+
+	\text{PAF}_{pop}=\frac{p_c(RR_{adj}-1)}{RR_{adj}}
```

Comment:
Instead of `PAF_pop`, should it just be `PAF`, since "P" stands for "population"?

Same comment for every other equation involving the PAF.